ext/dom/tests/DOMDocument_saveHTMLFile_formatOutput_gte_2_13.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+DOMDocument::saveHTMLFile() should format output on demand
+--CREDITS--
+Knut Urdalen <knut@php.net>
+#PHPTestFest2009 Norway 2009-06-09 \o/
```

Comment:
I'm confused by the credits section here?

Reply:
Copied over from the existing test, as almost everything is the same the credits still belong to this person. This test is a variant for libxml>=2.13.
Same for the other test.

Reply:
ACK